src/helpers/pivot/pivot_registry.ts

```diff
@@ -55,6 +55,6 @@ pivotRegistry.add("SPREADSHEET", {
   onIterationEndEvaluation: (pivot: SpreadsheetPivot) => pivot.markAsDirtyForEvaluation(),
   dateGranularities: [...dateGranularities],
   datetimeGranularities: [...dateGranularities, "hour_number", "minute_number", "second_number"],
-  isMeasureCandidate: (field: PivotField) => !["datetime", "boolean"].includes(field.type),
+  isMeasureCandidate: (field: PivotField) => field.type !== "boolean",
```

Comment:
Is the next step also allowing boolean measures ? Or is there  a specific reasons they don't work ?

Reply:
probably yes and yes